### PR TITLE
ENG-3606 amend coverage command

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by cecchisandrone** [comment]
@jeffgo10 please fix snyk vulnerability

File: .nycrc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "cache": false,
+  "check-coverage": true,
+  "extension": [
+    ".ts"
+  ],
+  "include": [
+    "test/**/*.ts"
```
Comment (cecchisandrone) [inline .nycrc]: coverage should be calculated on `src`, not `test`

File: package.json [hunk 2]
```diff
@@ -33,6 +33,11 @@
   "keywords": [
     "oclif"
   ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run coverage"
```
Comment (cecchisandrone) [inline package.json]: this shouldn't be here, move it to `.lintstagedrc`

**Review by cecchisandrone — CHANGES_REQUESTED** [review]

File: .nycrc (hunk 1 above)
Comment (jeffgo10) [reply .nycrc]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**